docs/experiments-msmarco-passage.md

```diff
@@ -539,4 +539,5 @@ The BM25 run with default parameters `k1=0.9`, `b=0.4` roughly corresponds to th
 + Results reproduced by [@ShreyasP20](https://github.com/ShreyasP20) on 2024-11-24 (commit [`c619dc8`](https://github.com/castorini/anserini/commit/c619dc8d9ab28298251964053a927906e9957f51))
 + Results reproduced by [@Divyajyoti02](https://github.com/Divyajyoti02) on 2024-11-24 (commit [`a1bcf88`](https://github.com/castorini/anserini/commit/a1bcf8853062da9f73915c873968a4e998d4e904))
 + Results reproduced by [@nihalmenon](https://github.com/nihalmenon) on 2024-11-29 (commit [`aa91e2db`](https://github.com/castorini/anserini/commit/aa91e2db224c752cc6a947bdf66bdbf9ce90af25))
-+ Results reproduced by [@sherloc512](https://github.com/sherloc512) on 2024-12-04 (commit [`9e55b1c`](https://github.com/castorini/anserini/commit/9e55b1c97fced46530dac1f78975d19635ffaf7a))
\ No newline at end of file
++ Results reproduced by [@sherloc512](https://github.com/sherloc512) on 2024-12-04 (commit [`9e55b1c`](https://github.com/castorini/anserini/commit/9e55b1c97fced46530dac1f78975d19635ffaf7a))
++ Results reproduced by [@zdann15](https://github.com/zdann15) on 2024-12-04 (commit [`4bea8e5`](https://github.com/castorini/anserini/commit/4bea8e59128a7ebf440b3ed5b4821a415b80a883))
```

Comment:
if you go to https://github.com/castorini/anserini/commit/4bea8e59128a7ebf440b3ed5b4821a415b80a883

you get "This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository."... so it's not referring to a valid commit in main trunk.

please fix...